Mathlib/Init/Data/Rat.lean

```diff
@@ -0,0 +1,15 @@
+/-
+Copyright (c) 2019 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro
+-/
+
+import Std.Data.Rat
+
+/-!
+# Notation for the rational numbers -/
+
+namespace Rat
+
+/-- Write the rational numbers as `ℚ`. -/
+notation "ℚ" => Rat
```

Comment:
```suggestion
@[inherit_doc] notation "ℚ" => Rat
```
we want people hovering on `ℚ` to see the documentation about `Rat`, not the documentation about the notation.